starter_code/hangman.js

```diff
@@ -20,16 +22,21 @@ class Hangman {
   }
 
   checkClickedLetters(key) {
-      if( !this.letters.includes(key) ){
-        this.letters.push(key.key)
-        if( this.secretWord.includes(key) ) this.addCorrectLetter(key)
-        return true
-      } 
+    if( this.checkIfLetter(key.keyCode )){
+      if( !this.letters.includes(key.key) ){
+        this.letters.push(key.key.toLocaleUpperCase())
+        
+        if( this.secretWord.includes(key.key.toLocaleUpperCase()) ) {
+          this.addCorrectLetter(key.key.toLocaleUpperCase())
+          return true
+        }
+        this.addWrongLetter(key.key)
+      } }
     return false
   }
 
   addCorrectLetter(i) {
```

Comment:
Algo asi podria haber funcionado, ya que es mas facil traer la variable de secretword y asignarla a una nueva para poder manipularla, separarla letra por letra y asi las letras que el usuario este agregando irlas metiendo al nuevo array en caso de ser correctas.

 ` addCorrectLetter(i) {`
  `  let newstr = this.secretWord.split("")`
`    this.guessedLetter += newstr[i].toLocaleUpperCase()`
`    return this.checkWinner()`
 ` }`